components/lib/crash/src/main/java/mozilla/components/lib/crash/service/MozillaSocorroService.kt

```diff
@@ -55,11 +62,27 @@ class MozillaSocorroService(
     private val version: String = BuildConfig.MOZILLA_VERSION,
     private val buildId: String = BuildConfig.MOZ_APP_BUILDID,
     private val vendor: String = BuildConfig.MOZ_APP_VENDOR,
-    private val serverUrl: String = "https://crash-reports.mozilla.com/submit?id=$appId&version=$version&$buildId"
+    private var serverUrl: String? = null,
+    private var versionName: String = "0.0"
 ) : CrashReporterService {
     private val logger = Logger("mozac/MozillaSocorroCrashHelperService")
     private val startTime = System.currentTimeMillis()
 
+    init {
+        try {
+            versionName = applicationContext.packageManager
+                .getPackageInfo(applicationContext.packageName, 0).versionName
+        } catch (e: PackageManager.NameNotFoundException) {
+            Logger.error("package name not found, failed to get application version")
+        } catch (e: IllegalStateException) {
+            Logger.error("failed to get application version")
+        }
+
+        if (serverUrl == null) {
+            serverUrl = "https://crash-reports.mozilla.com/submit?id=$appId&version=$versionName"
+        }
```

Comment:
How will that work on crash-stats which seems to only show Gecko versions?

Reply:
The collector ignores querystring parameters in the crash report submission, but they sometimes help people to debug issues.

Reply:
Nice. In this case maybe consider adding the AC version additionally (which is maybe more interesting than the app version if we need to debug something regarding crash reports), @rocketsroger?

Reply:
Agreed.  Updated `serverUrl`